tests/test_pytorch/test_accelerator.py

```diff
@@ -465,17 +465,17 @@ def test_reduce_op_strategy(tmpdir, reduce_op, expectation):
 
 
 @pytest.mark.skip(reason="TBD : Fix pytest issues")
-@pytest.mark.skipif(HPUAccelerator.auto_device_count() < 8, reason="Test requires multiple HPU devices")
+@pytest.mark.skipif(HPUAccelerator.auto_device_count() < 2, reason="Test requires multiple HPU devices")
 @pytest.mark.parametrize(
     ("reduce_op", "logged_value_epoch", "logged_value_step"),
     [
-        # Epoch = Sum(42, 43, 44) * 8, Step = 44 * 8 (for 8 ddp processes)
+        # Epoch = Sum(42, 43, 44) * 2, Step = 44 * 2 (for 2 ddp processes)
         ("sum", 1032.0, 352.0),
-        # Epoch = Max(42, 43, 44), Step = Max(44, ... (x8))
+        # Epoch = Max(42, 43, 44), Step = Max(44, ... (x2))
         ("max", 44.0, 44.0),
-        # Epoch = Min(42, 43, 44), Step = Min(44, ... (x8))
+        # Epoch = Min(42, 43, 44), Step = Min(44, ... (x2))
         ("min", 42.0, 44.0),
-        # Epoch = Mean(42(x8), 43(x8), 44(x8)), Step = Mean(44, ... (x8))
+        # Epoch = Mean(42(x2), 43(x2), 44(x2)), Step = Mean(44, ... (x2))
```

Comment:
not sure what was the x8 and now x2 for?

Reply:
x8 referred to number of cards. 
This change will not affect max, min and mean values.
sum values will change to 258, 88